Utils/trigger_content_non_ami_build.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+_branch=$1
+_circle_token=$2
+
+trigger_build_url=https://circleci.com/api/v1/project/demisto/content/tree/${_branch}?circle-token=${_circle_token}
+
+if [ -z "$3"]
+  then
+    post_data=$(cat <<EOF
+    {
+      "build_parameters": {
+        "AMI_RUN": "true"
```

Comment:
should AMI_RUN be true in both cases?

Reply:
Actually it should be false, but on both cases